conf/application.conf

```diff
@@ -304,7 +303,9 @@ auth {
 }
 
 time-machine {
-    add-years = 1
+    enable-time-machine = false
```

Comment:
not sure why we moved enable-time-machine under time-machine section?
this is the same FS as the rest of the FSs we have so far, no need to introduce this complexity

Reply:
Because it's not going to be on the FS list on the FS page I thought to make it a bit separate. Do you thinks it's better to move it to where all the other FS are?